components/dashboard/src/feedback-form/FeedbackComponent.tsx

```diff
@@ -0,0 +1,160 @@
+        return (
+            <>
+                <button
+                    className={
+                        "hover:scale-150 transform bg-transparent bottom-5 right-5 cursor-pointer " +
+                        (selectedEmoji === emojiScore["starry"] ? "" : "grayed")
+                    }
+                    onClick={(e) => handleClick(e.target)}
+                >
+                    <img src={starryEyed} alt="starry eyed emoji" width={width || "24px"} title="starry" />
+                </button>
+                <button
+                    className={
+                        "hover:scale-150 transform bg-transparent bottom-5 right-5 cursor-pointer " +
+                        (selectedEmoji === emojiScore["happy"] ? "" : "grayed")
+                    }
+                    onClick={(e) => handleClick(e.target)}
+                >
+                    <img
+                        className="bottom-5 right-5 cursor-pointer"
+                        title="happy"
+                        src={happy}
+                        alt="happy emoji"
+                        width={width}
+                    />
+                </button>
+                <button
+                    className={
+                        "hover:scale-150 transform bg-transparent bottom-5 right-5 cursor-pointer " +
+                        (selectedEmoji === emojiScore["meh"] ? "" : "grayed")
+                    }
+                    onClick={(e) => handleClick(e.target)}
+                >
+                    <img
+                        className="bottom-5 right-5 cursor-pointer"
+                        title="meh"
+                        src={meh}
+                        alt="meh emoji"
+                        width={width}
+                    />
+                </button>
+                <button
+                    className={
+                        "hover:scale-150 transform bg-transparent bottom-5 right-5 cursor-pointer " +
+                        (selectedEmoji === emojiScore["crying"] ? "" : "grayed")
+                    }
+                    onClick={(e) => handleClick(e.target)}
+                >
+                    <img
+                        className="bottom-5 right-5 cursor-pointer"
+                        title="crying"
+                        src={crying}
+                        alt="crying emoji"
+                        width={width}
+                    />
+                </button>
+            </>
+        );
+    };
+    return (
+        <>
+            <h3 className="mb-4">Send Feedback</h3>
+            {selectedEmoji ? (
+                <>
+                    <div className="flex flex-col -mx-6 px-6 py-4 border-t border-b border-gray-200 dark:border-gray-800 bg-white dark:bg-gray-900">
+                        <div className="relative">
+                            <div className="absolute flex bottom-5 right-5">{emojiGroup(24)}</div>
+                            <textarea
```

Comment:
**issue**: Re-posting here from https://github.com/gitpod-io/gitpod/pull/10061#discussion_r875774731 as this is still an issue before 8292fef, and resolving the original comment. Could we use border color for default and focus state as in other text inputs?

Reply:
@laushinka The border colors have been fixed for the light theme! This still needs the textarea :a: background and :b: border color on default and hover state to follow the designs of other text inputs in dark theme. To clarify, I'm reposting some screenshots below. 🌔 

| Feedback | Variable |
|-|-|
| <img width="559" alt="Screenshot 2022-05-19 at 2 00 27 PM" src="https://user-images.githubusercontent.com/120486/169278607-291312ee-fc88-4a5d-9a42-28a811b02702.png"> | <img width="555" alt="Screenshot 2022-05-19 at 2 00 37 PM" src="https://user-images.githubusercontent.com/120486/169278612-47e381e3-5def-4252-b96a-9544b08eccf5.png"> |

Reply:
@gtsiolis I just pushed a fix for the textarea background color, but couldn't figure out the difference in the border colors, so I think I need your help here to tell me where to add which classnames? 🙏🏽

Reply:
Done, thanks!

Reply:
Thanks, @laushinka! Looks good, left one last UX comment below in https://github.com/gitpod-io/gitpod/pull/10061#discussion_r877066717. This is almost at the finish line! 🏁